<launch>
  <node name="static_door_publisher_node" pkg="tuw_door_detection" type="static_door_publisher_node" output="screen">
  </node>
</launch>
